pyworklistserver/worklist_server.py

```diff
@@ -1,40 +1,66 @@
 """ DICOM server implementation that responds to C-FIND requests"""
+import random
+import os
+import time
 from pynetdicom import AE
 from pynetdicom import evt
 from pynetdicom.sop_class import ModalityWorklistInformationFind
 
 from pyworklistserver import worklist
+from pyworklistserver import user_config
+from pyworklistserver import get_config
 
 class WorklistServer:
     """ Simple worklist server with support for one client. This
     implementation class is responsible for network communication with
     the client. """
 
-    def __init__(self, serverConfig, app_logger, seed, blocking):
+    def __init__(self, serverConfig, app_logger, blocking):
         self._config = serverConfig
+        self._config_values = get_config.GetConfig()
         self._logger = app_logger
         self._blocking = blocking
-        self._seed = seed
-        self._worklist_factory = worklist.RandomWorklist('ISO_IR 100', self._seed)
+        self._worklist_factory = worklist.RandomWorklist('ISO_IR 100', self._config_values)
         self._handlers = [
             (evt.EVT_C_FIND, self._on_find, [app_logger]),
         ]
         self._server = None
 
+    def get_seedNumber(self):

```

Comment:
This code should be changed in a follow-up PR to bring back the original --seed argument.

Reply:
fixed on issue #7